Mathlib/Data/Rel.lean

```diff
@@ -425,7 +425,7 @@ lemma exists_eq_singleton_of_prod_subset_id {s t : Set α} (hs : s.Nonempty) (ht
 
 variable (R) in
 /-- A relation `R` is symmetric if `a ~[R] b ↔ b ~[R] a`. -/
-protected abbrev IsSymm : Prop := IsSymm α (· ~[R] ·)
+protected abbrev IsSymm : Prop := Std.Symm (· ~[R] ·)
```

Comment:
Should this name be updated? If so, can you leave a TODO?

Reply:
I don't think it should, but not sure. What name do you suggest?

Reply:
I was thinking about `Symm`, but I guess that would go against naming guidelines. So `IsSymm` is fine.